MvvmCross.Android.Support/V7.AppCompat/MvxAppCompatViewPresenter.cs

```diff
@@ -429,11 +433,12 @@ protected virtual void OnFragmentPopped(FragmentTransaction ft, Fragment fragmen
             }
             else
                 throw new MvxException("ViewPager or TabLayout not found");
+            return Task.FromResult(true);
         }
         #endregion
 
         #region Close implementations
-        protected override bool CloseFragmentDialog(IMvxViewModel viewModel, MvxDialogFragmentPresentationAttribute attribute)
+        protected override async Task<bool> CloseFragmentDialog(IMvxViewModel viewModel, MvxDialogFragmentPresentationAttribute attribute)
```

Comment:
Can we please not use the `async` keyword if we are not awaiting anything inside of the body?

Reply:
Will fix

Reply:
Done

Reply:
Thanks 👍